crates/matrix-sdk-ui/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ All notable changes to this project will be documented in this file.
 
 - [**breaking**] Optionally allow starting threads with `Timeline::send_reply`.
   ([4819](https://github.com/matrix-org/matrix-rust-sdk/pull/4819))
+- [**breaking**] Push the reply logic down into matrix_sdk.
```

Comment:
Agree it's a breaking change, but can you comment to which extent? Ideally we'd include all the names of types that have been changed here, but at least having some steps explaining how to migrate would be sweet :)

Reply:
Yes, true. Have expanded it.